docs/concepts.md

```diff
@@ -1,67 +1,69 @@
 # Concepts
 
-- [Runbook](#runbook)
-- [Node and Environment](#node-and-environment)
-- [Tools and Scripts](#tools-and-scripts)
-- [Platform](#platform)
-- [Feature](#feature)
-- [Requirement and Capability](#requirement-and-capability)
-- [Test suite and test case](#test-suite-and-test-case)
+- [Concepts](#concepts)
+  - [Test suite and test case](#test-suite-and-test-case)
+  - [Runbook](#runbook)
+  - [Tools and Scripts](#tools-and-scripts)
+  - [Node and Environment](#node-and-environment)
+  - [Platform](#platform)
+  - [Feature](#feature)
+  - [Requirement and Capability](#requirement-and-capability)
+
+
+## Test suite and test case
+
+A test suite contains one or more test cases that focus on certain functions or tested domains. A test case is the smallest unit that runs and generates test results.
+
+The test cases in the same test suite have the same overall description and setting/clearing method. Each test case describes its own steps and sets the appropriate priority.
+
+Test suites and test cases have their metadata. The metadata includes the description, priority, test requirement, and other information.
 
 ## Runbook
 
-A runbook contains all the configurations of LISA operation, so it is called the runbook. With Runbook, the command line is very simply.
+The runbook contains all the configurations of LISA operation. It keeps you from lengthy command-line commands and makes it easy to adjust configurations.
 
-The previous version of LISA is powerful and supports many scenarios. But this increases the complexity of the command-line parameters and relies on a secret file. The current LISA provides a consistent way to manage configuration and maximize customization capabilities.
+The previous version of LISA is powerful and supports many scenarios. However, the trade-off there is increased complexity of the command-line parameters, and it relies on a secret file to work. The current version of LISA provides a consistent way to manage configuration and maximize customization capabilities through a runbook.
 
-One runbook can refer to other runbooks so that test case selection, platform configuration and other options can be separated. It helps reduce duplication.
+One runbook can refer to other runbooks so that test case selection, platform configuration and other options can be separated. It helps reduce redundancy.
 
 The runbook supports custom variables, and these variables can be provided in the runbook, command line or environment variables. Variables are only used in the runbook and are resolved to actual values before the start of the test.
 
 The configuration of the runbook can be extended to certain components. For example, when defining a notifier, you can define its configuration structure together. The corresponding configuration will be loaded into the notifier with the same schema. The extended configuration also supports variables.
 
-## Node and Environment
+## Tools and Scripts
```

Comment:
What's the reason change order of "tool" and "node"? In my mind, the node is a prior concept before tools. The tools are installed on nodes. The content of tools mentions node many times.

Reply:
Indeed. I agree that node is like a big prerequisite to all concepts, but I was trying to classify something to runnable(utilities)/unrunnable(design concepts), and I put runnables at first as I thought a test case would be a basis. I think I will reverse the order so the design concepts go first.

Reply:
If users run our tests, they may need to know node/environment. Some time, users may be care about test infra, how many virtual machines are created. If users needs to write test cases, they need to know both node and tools. The tools is how code organized underlying.